spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/main/java/org/springframework/boot/configurationprocessor/fieldvalues/FieldValuesParser.java

```diff
@@ -29,6 +29,7 @@
  * @author Phillip Webb
  * @since 1.1.2
  * @see JavaCompilerFieldValuesParser
+ * @see UnresolvableDefaultValue
```

Comment:
I only edited this to show the intention to add some default methods to set and inspect if a field was unresolvable to "hide" the `UnresolvableDefaultValue` impl detail.